docs/CONTRIBUTING.md

```diff
@@ -58,6 +58,7 @@ which can be installed by running `xcode-select --install`.
 
 #### Linux
 
+- npm 3.10 is required (install with `npm install --global npm@3.10` for example). We cannot guarantee proper installation with other npm versions.
```

Comment:
Also the node v6 & npm v3 requirement is for all platforms, so I think it'd be better to mention it under common prerequisites on line 20

Reply:
@jhermsmeier do you want me to mention the node v6 as well here?